zpi-frontend/src/layout/Navigation.tsx

```diff
@@ -111,76 +112,79 @@ const Navigation = ({ children }: NavigationProps) => {
-                                                    </Dropdown.Menu>
-                                                </Dropdown>
-
-                                </li>
-                                        ) : null}
-                                        {user?.roles?.some((role: Role) => role.name === 'supervisor') ? (
+                <div className={`container p-0 ${isLoginPage ? 'd-none' : ''}`}>
+                    <nav className="navbar navbar-expand-lg navbar-light bg-light">
+                        <div className="container">
+                            <Link className="navbar-brand" to="/">ZPI Helper</Link>
+                            <button className="navbar-toggler" type="button" onClick={() => setShowNav(!showNav)}>
+                                <span className="navbar-toggler-icon"></span>
+                            </button>
+                            <div className={`collapse navbar-collapse ${showNav ? 'show' : ''}`} id="navbarNav">
+                                <ul className="navbar-nav me-auto">
+                                    <li className="nav-item">
+                                        <NavLink className={({ isActive }) => isActive ?
+                                            "nav-link active" : "nav-link"} to="/">{t('navigation.home')}</NavLink>
+                                    </li>
+                                    {isLoggedIn ? (
+                                        <>
                                             <li className="nav-item">
                                                 <NavLink className={({ isActive }) => isActive ?
-                                                    "nav-link active" : "nav-link"} to="/my">
-                                                    {t('navigation.myTheses')}
+                                                    "nav-link active" : "nav-link"} to="/theses" >
+                                                    {t('general.university.theses')}
                                                 </NavLink>
                                             </li>
-                                        ) : null}
-                                    </>
-                                ) : null}
-                            </ul>
+                                            {user?.roles?.some((role: Role) => role.name === 'admin') ? (
+                                                <li className="nav-item">
+                                                    <Dropdown as={Nav.Item}>
+                                                        <Dropdown.Toggle as={Nav.Link} className={isManagementActive ? "active" : ""}>{t('navigation.manage')}</Dropdown.Toggle>
+                                                        <Dropdown.Menu>
+                                                            <Dropdown.Item as={Link} to="/thesis" className={location.pathname === '/thesis' ? "active" : ""}>
```

Comment:
Reszta endpointów w tym dropdownie jest w liczbie mnogiej. Może tutaj też fajnie byłoby dać `/theses`, a dla ogólnodostępnych tematów na przykład `public-theses` lub coś w tym stylu?

Reply:
Moge zmienić na coś takiego. Dałam thesis właśnie dlatego, że te ogólne tematy mają theses i nie wiedziałam na co by to zmienić.